server/src/milvus/milvus.service.ts

```diff
@@ -55,6 +55,10 @@ export class MilvusService {
         if (process.env.SERVER_NAME) {
           clientConfig.tls.serverName = process.env.SERVER_NAME;
         }
+
+        if (process.env.DB_NAME) {
```

Comment:
I don't think you need this here. you can set the `process.env.DATABASE`. it will be passed from the client. 

But still, you need to pass the database to the MilvusClient. 

like this：

```javascript
const clientConfig: ClientConfig = {
        address: milvusAddress,
        token,
        username,
        password,
        logLevel: process.env.ATTU_LOG_LEVEL || 'info',
        database: database || this.DEFAULT_DATABASE,
      };
```

Reply:
Thanks for the patch.

Reply:
> I don't think you need this here. you can set the `process.env.DATABASE`. it will be passed from the client.
> 
> But still, you need to pass the database to the MilvusClient.
> 
> like this：
> 
> ```js
> const clientConfig: ClientConfig = {
>         address: milvusAddress,
>         token,
>         username,
>         password,
>         logLevel: process.env.ATTU_LOG_LEVEL || 'info',
>         database: database || this.DEFAULT_DATABASE,
>       };
> ```

hi @shanghaikid , didn't get this.
i am assigning this to `clientConfig.database` inside if statement. i see in SDK that clientConfig has a optional field database. it is also working this way.
I did not understood your comment here

Reply:
`database` should be passed from the client to the server side.  
The client will read `process.env.DATABASE` .

Yes, you can setup new `DB_NAME` env here, but it will be a conflict with `process.env.DATABASE`

Reply:
got it, added.
thanks @shanghaikid